Add node name details in mayactl volume info command

slalwani97 opened the pull request:
Signed-off-by: Sumit Lalwani <sumit.lalwani97@gmail.com>

<!--  Thanks for sending a pull request!  Here are some tips for you -->

**What this PR does / why we need it**:
* Shows the volume replicas detail with the node name in which that volume is running.
* Shows volume name in mayactl volume info command.
* Adds extra column for Node in info details.

**Which issue this PR fixes**: fixes #350 

**Special notes for your reviewer**:
![screenshot from 2018-06-04 12-25-39](https://user-images.githubusercontent.com/29499601/40975305-1a75e66c-68e8-11e8-9763-f873be1d9879.png)

prateekpandey14 commented on cmd/mayactl/app/command/volume_info.go:
```diff
@@ -110,11 +143,20 @@ func (c *CmdVolumeInfoOptions) DisplayVolumeInfo(a *Annotations, collection clie
+
+		replicaTemplateWithNodeName = `
+================= Replica Details ==================================
+IP            AccessMode               Status           Node
```
@slalwani97  columns are not aligned properly as i can see in the output ?

slalwani97 commented:
![screenshot from 2018-06-01 18-23-49](https://user-images.githubusercontent.com/29499601/40842195-b100b0d8-65ca-11e8-83a8-c9f61584ecbe.png)

fixed the alignment and removed flag

kmova commented:
@slalwani97 This PR is coming nicely. The alignment could still be an issue. Can you try to run this command on a volume that is generated by PVC?

slalwani97 commented:
@kmova Sir, yes alignment was an issue because volume names can be of any length. Now I have used tabwriter with template to align properly.

slalwani97 commented:
![screenshot from 2018-06-04 01-43-06](https://user-images.githubusercontent.com/29499601/40890764-cb35dff8-6798-11e8-87aa-69bf617700fb.png)

NisantaSahoo commented:
******Star PR **********
